demo_content/content/tide_grant/tc-9b-grant.yml

```diff
@@ -0,0 +1,216 @@
+          field_paragraph_accordion_name: 'Assessment criteria'
+          field_paragraph_accordion_body: 'Lorem ipsum'
+        - entity: paragraph
+          type: accordion_content
+          field_paragraph_accordion_name: 'Assessment process'
+          field_paragraph_accordion_body: 'Lorem ipsum'
+        - entity: paragraph
+          type: accordion_content
+          field_paragraph_accordion_name: 'How to apply'
+          field_paragraph_accordion_body: 'Lorem ipsum'
+  field_node_on_going: false
+  field_node_documents: 
+    - '#process':
+        callback: reference
+        args:
+          - media
+          - bundle: document
+            name: 'Demo Sample Document'
+  field_node_timeline: 
+    - entity: paragraph
+      type: timelines
+      field_paragraph_title: Demo Timeline
+      field_timeline:
+        - entity: paragraph
+          type: timeline
+          field_paragraph_title: Demo Timeline Item
+          field_paragraph_date_range:
+            - value: '2020-01-01T08:00:00'
+              end_value: '2020-01-01T10:00:00'
+          field_paragraph_link: https://www.google.com
+          field_paragraph_media:
+            - '#process':
+                callback: reference
+                args:
+                  - media
+                  - bundle: image
+                    name: 'Demo: Melbourne tram'
+          field_paragraph_summary: Mauris tincidunt tincidunt felis vel tempus. Vestibulum rhoncus blandit justo quis finibus.
+        - entity: paragraph
+          type: timeline
+          field_paragraph_title: Demo Timeline Item 2
+          field_paragraph_cta_text: Alternative
+          field_paragraph_link: https://www.google.com
+          field_paragraph_media:
+            - '#process':
+                callback: reference
+                args:
+                  - media
+                  - bundle: image
+                    name: 'Demo: Melbourne tram'
+          field_paragraph_summary: Anim excepteur magna eu aliqua quis velit voluptate sit culpa.
+  # Other.
+  field_show_social_sharing: 1
+  field_show_content_rating: 1
+  field_show_table_of_content: 1
+  field_topic:
+    - '#process':
+        callback: reference
+        args:
+          - taxonomy_term
+          - vid: topic
+            name: 'Demo Topic'
+  field_tags:
+    - '#process':
+        callback: reference
+        args:
+          - taxonomy_term
+          - vid: tags
+            name: 'Demo Tag'
+  # Sites.
+  field_node_site:
+    - '#process':
+        callback: reference
+        args:
+          - taxonomy_term
+          - vid: sites
+            name: 'Demo Site'
+    - '#process':
+        callback: reference
+        args:
+          - taxonomy_term
+          - vid: sites
+            name: 'aboriginalvictoria.vic.gov.au'
+    - '#process':
+        callback: reference
+        args:
+          - taxonomy_term
+          - vid: sites
+            name: 'aboriginalheritagecouncil.vic.gov.au'
```

Comment:
So these are static? Will they need to be updated every time we add a new semi independant site?

Reply:
Yes, I couldn't find anything which will run kind of a loop in yml content. So in that way a function could have grabbed all the site names and then add it here. Let me know if you have any ideas, I will try that.

Reply:
@MdNadimHossain It's worth investigating entity reference processing using yaml content. See this blog post for an example [Creating Content with YAML Content Module | Mediacurrent Blog Post](https://www.mediacurrent.com/blog/creating-content-yaml-content-module)

Reply:
@GROwen Yes, I was just following this link which has the same explanation - [Referencing Entities: Adding Tags](https://www.drupal.org/docs/8/modules/yaml-content/examples-and-snippets-v8x-1x#s-referencingentities-adding-tags)
In the yml content, I am calling the processor multiple times for the same field but for multiple values.